distribution/docs/src/main/resources/content/_running/command-scheduler.adoc

```diff
@@ -16,24 +16,30 @@ Writing these types of scripts are specific to the administrator's operating sys
 The administrator can also create a Command Schedule, which currently requires only two fields.
 The Command Scheduler only runs when the container is running, so there is no need to verify if the container is up.
 In addition, when the container is restarted, the commands are rescheduled and executed again.
+A command will be repeated executed indefinitely according to the configured interval until the container is shutdown or the Scheduled Command is deleted.
```

Comment:
repeatedly?

Reply:
Fixed.

Also, @clockard, I added a commit to add a catch and log message for an exception that happens when systems are slow to start up, so slow that a 1 minute delay isn't enough. This should prevent that ugly error message.

Reply:
@peterhuffer, if a command fails, it will be attempted again at the next interval (this is included in the documentation). I did investigate adding some retry logic: https://github.com/codice/ddf/pull/2357. Even when the subject is available, it could fail for other security-related reasons. I'm not sure exactly what we need to wait for before running the command.